pandas/core/internals/managers.py

```diff
@@ -1505,7 +1505,7 @@ def get_slice(self, slobj, axis=0):
         if axis >= self.ndim:
             raise IndexError("Requested axis not found in manager")
 
-        return type(self)(self._block._slice(slobj), self.index[slobj], fastpath=True)
+        return type(self)(self._block._slice(slobj), self.index[slobj], fastpath=True,)
```

Comment:
i think we dont want the trailing comma here

Reply:
Worth rerunning CI over?

Reply:
i can change it in my next "assorted cleanups" PR